thunderbird-development/building-thunderbird/linux-build-prerequisites.md

```diff
@@ -4,41 +4,89 @@ description: >-
   environment set up and ready to hack on Thunderbird.
 ---
 
-# Linux Build Prerequisites
-
-This guide assumes you already followed the [Getting Started](../getting-started.md) documentation and you already downloaded `mozilla-central` and `comm-central` source code.
+# Hardware Requirements
 
 ## 64-bit version
 
-You need a 64 bit version of Linux in order to build Thunderbird. You can check which version you're running by typing this command in your terminal:
+You will need to be running a 64-bit version of Linux in order to build Thunderbird. You can check which version you're running by typing this command in your terminal:
 
 ```
 uname -m
 ```
 
 if this command returns `x86_64` you can proceed.
 
+## 30 GB of free space
+
+The Thunderbird build will take up to 30 GB of disk space in order to complete. Be sure to have enough free space and a fast internet connection to avoid interruptions.
```

Comment:
nit: It's not technically required to have an internet connection to build. The build itself is intended to work without going out to the internet at all. However, the default when building from mozilla/comm-central is that `--enable-bootstrap` is set so that the toolchains download automatically.

Not a big deal, but the broad statement of requiring an internet connection may ruffle feathers.

Reply:
great point. I'll add a hint with this info. Where is the `--enable-bootstrap` set so I can tell people without internet where to go and remove this?